lib/workers/repository/update/pr/index.ts

```diff
@@ -323,6 +335,7 @@ export async function ensurePr(
     {
       debugData: updatePrDebugData(
         config.baseBranch,
+        prepareLabels(config).sort(), // include labels in debug data, sort them just to be safe
```

Comment:
Does it work to have the prepareLabels() function also do the sorting?

Reply:
Yes.